test/helpers/client-stub.js

```diff
@@ -1,11 +1,9 @@
-const mentions = require('./mentions');
+const UserStub = require('./user-stub');
```

Comment:
I think I'd rather use the official classes for tests. Trying to think of how best to do that